src/cpp/rtps/common/Time_t.cpp

```diff
@@ -61,6 +61,9 @@ static void current_time_since_unix_epoch(
 namespace eprosima {
 namespace fastrtps {
 
+constexpr int32_t Time_t::INFINITE_SECONDS;
+constexpr uint32_t Time_t::INFINITE_NANOSECONDS;
+
```

Comment:
Why are these necessary? Are they not obtained from `#include <fastdds/rtps/common/Time_t.h>` ?

Reply:
Until C++17, `static constexpr` have to be defined, not only declared. In this CPP I am defining the variables.